src/dataobjects.py

```diff
@@ -78,7 +78,7 @@ def __init__(self, data: dict):
         self.table = None
 
     def _extract_attributes(self, data: dict):
-        self.guid = data.get("warehouse_table")
+        self.guid = data.get("warehouse_table").get("guid")
```

Comment:
this will throw if there is no warehouse table, that should be handled, if we don't care about this then I would prefer if you used indexing syntax to signal that we always expect to see `warehouse_table` since it will generate a better error message in case something does go wrong

Reply:
If there is no warehouse table it wont create this instance. The function that fetches from the API does this:

https://github.com/selectstar/dbt-impact-report-action/blob/e575b45a4ad6d994ac8a620f93d1f23eff3c8ee3/src/selectstar.py#L99

Reply:
Changed the syntax for the sake clearer message

Reply:
Thanks!